### ISSUE TITLE
Items with 3D Models Render Inside the Drawer

### ISSUE THREAD

**Issue by Avakining** [Buuz135/FunctionalStorage#122]
This affects all items with non-cube 3D models, like Immersive Engineering's Cloches, any item patched by [Vanilla Tweaks](vanillatweaks.net)' 'Item Stitching Fix' resource pack, Botania's Petal Apothecary and Mana Blaster, and almost everything from Create. See screenshots:

<img width="1512" alt="Screenshot 2022-11-15 at 15 57 19" src="https://user-images.githubusercontent.com/25210618/202023972-35770dd4-bc9f-4dc5-b3a0-207b45106df5.png">

View from inside the drawer in Spectator mode:
<img width="1512" alt="Screenshot 2022-11-15 at 15 57 52" src="https://user-images.githubusercontent.com/25210618/202024053-69d6f967-4f7a-49de-a47d-9380e80b5e7e.png">
<img width="1512" alt="Screenshot 2022-11-15 at 15 57 48" src="https://user-images.githubusercontent.com/25210618/202024055-c5bda528-ab88-4c0d-a60f-ef6fd6215579.png">

Forge 40.1.84
Titanium 3.5.6
Functional Storage 1.18.2-1.0.7

**Comment by vvaffaru** [comment]
After installing some mods for a modpack I'm working on, this started happening to some seemingly random items. From what I've seen so far in my world, clay balls, lapis, coal, and flint are all rendering inside the drawer, so I'm not sure if it's just items with 3D models being affected by this bug. (Unless that's what those items are.) I disabled all the mods I added and the issue persists. Not sure how to fix it on my end.

Here's a [mod list.](https://docs.google.com/spreadsheets/d/1nbeLbdskefuNDdPKg--oTAJatUykdsgx2B8aS0VlNFU/edit?usp=sharing) All the entries with "disabled" in the A column were added today. Everything was working fine before adding all the mods that are currently disabled.

Forge 43.1.57
Functional Storage 1.19.2-1.0.7
